pkg/driver/modbus/client.go

```diff
@@ -14,17 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package driver
+package modbus
 
 import (
 	"errors"
 	"sync"
 	"time"
 
+	"github.com/kubeedge/mappers-go/pkg/common"
```

Comment:
the order of import packages should be optimized.

Reply:
done